documents/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,1243 @@
+    return
+
+JSONPayload = '{"state":{"desired":{"property":' + str(3) + '}}}'
+deviceShadowHandler.shadowUpdate(
+        JSONPayload,
+        customShadowCallback_Update,
+        5)
+
+```
+
+#### Example of updating a Classic Shadow in the v2 SDK
+
+```python
+# Update shadow
+def on_update_shadow_accepted(response):
+    return
+ update_accepted_subscribed_future, _ =
+        shadow_client.subscribe_to_update_shadow_accepted(
+                request=iotshadow.UpdateShadowSubscriptionRequest(
+                        thing_name=shadow_thing_name),
+                qos=mqtt5.QoS.AT_LEAST_ONCE,
+                callback=on_update_shadow_accepted)
+
+def on_update_shadow_rejected(error):
+    return
+update_rejected_subscribed_future, _ =
+        shadow_client.subscribe_to_update_shadow_rejected(
+                request=iotshadow.UpdateShadowSubscriptionRequest(
+                        thing_name=shadow_thing_name),
+                qos=mqtt5.QoS.AT_LEAST_ONCE,
+                callback=on_update_shadow_rejected)
+
+request = iotshadow.UpdateShadowRequest(
+        thing_name="thing name",
+        state=iotshadow.ShadowState(
+                reported={color: 1},
+                desired={color: 2},),
+        client_token="<token>")
+
+future = shadow_client.publish_update_shadow(
+        request,
+        mqtt5.QoS.AT_LEAST_ONCE)
+
+```
+
+#### Example of subscribing to a delta Shadow events in the v1 SDK
+
+```python
+# Delta Events
+shadow_client.connect()
+
+def customShadowCallback_Delta(payload, responseStatus, token):
+    return
+
+deviceShadowHandler = shadow_client.createShadowHandlerWithName(
+        "<thing name>",
+        True)
+
+deviceShadowHandler.shadowRegisterDeltaCallback(
+        customShadowCallback_Delta)
+
+```
+
+#### Example of subscribing to a delta Shadow events in the v2 SDK
+
+```python
+# Delta Events
+def on_shadow_delta_updated(delta):
+    return
+
+delta_subscribed_future, _ = shadow_client.subscribe_to_shadow_delta_updated_events(
+        request=iotshadow.ShadowDeltaUpdatedSubscriptionRequest(
+                thing_name=shadow_thing_name),
+        qos=mqtt5.QoS.AT_LEAST_ONCE,
+        callback=on_shadow_delta_updated)
+
+delta_subscribed_future.result()
+
+```
+
+For more information, see API documentation for the v2 SDK
+[Device Shadow](https://aws.github.io/aws-iot-device-sdk-python-v2/awsiot/iotshadow.html).
+
+For code examples, see the v2 SDK [Device Shadow](https://github.com/aws/aws-iot-device-sdk-python-v2/blob/main/samples/shadow_mqtt5.py).
+
+
+### Client for AWS IoT Jobs
+
+The v1 SDK and the v2 SDK offer support of AWS IoT Core services implementing a service client for the
+[Jobs](https://docs.aws.amazon.com/iot/latest/developerguide/iot-jobs.html) service which helps with defining a set of
+remote operations that can be sent to and run on one or more devices connected to AWS IoT.
+
+
+The v2 SDK supports Jobs service as well, but with completely different API.
```

Comment:
The previous paragraph mentions both v1 and v2, so the phrase "The v2 SDK supports Jobs service as well" looks strange.